editor/src/uuiui/inputs/number-input.tsx

```diff
@@ -633,39 +649,30 @@ export const NumberInput = betterReactMemo<NumberInputProps>(
               className='number-input-innerLabel'
               css={{
                 position: 'absolute',
-                top: 1,
-                right: 1,
+                top: 0,
+                left: 0,
                 userSelect: 'none',
                 pointerEvents: 'none',
                 width: 20,
                 height: 20,
-                backgroundImage: backgroundImage,
-                borderRadius: UtopiaTheme.inputBorderRadius,
                 display: 'block',
-                '.number-input-container:hover &': {
-                  display: incrementControls && controlStyles.interactive ? 'none' : 'block',
-                },
               }}
             >
               <div
                 style={{
                   position: 'absolute',
                   pointerEvents: 'none',
-                  right: 0,
-                  top: 5,
+                  left: 0,
+                  top: 2,
                   textAlign: 'center',
-                  fontWeight: 'bold',
+                  fontWeight: 600,
                   fontSize: '9px',
                   width: '100%',
                   height: '100%',
                   color: controlStyles.secondaryColor,
                 }}
               >
-                {typeof labelInner === 'object' && 'type' in labelInner ? (
-                  <Icn {...labelInner} />
-                ) : (
-                  labelInner
-                )}
+                {labelInner}
```

Comment:
I believe this could throw errors when a label is containing iconprops

Reply:
Oh my. Does anyone recall the no doubt perfectly legitimate reason _why_ labelInner could be a props object?

Reply:
It is used now for different color pickers as an object prop, I think it can be safely changed to icon.

Reply:
Fixed by restoring the old code for now.